examples/triton/docker-compose-with-web.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 4g
+    restart: always
+    labels:
+      - triton.cns.services=redis
+    environment:
+      - CONTAINERPILOT=file:///etc/containerpilot.json
+      - affinity:com.docker.compose.service!=~redis
+    env_file: _env
+    network_mode: bridge
+
+  consul:
+    image: autopilotpattern/consul:0.7.2-r0.8
+    command: >
+      /usr/local/bin/containerpilot
+      /bin/consul agent -server
+        -bootstrap-expect 1
```

Comment:
Maybe add a comment about how to switch this so it runs as an HA, multi-node cluster, as in https://github.com/autopilotpattern/wordpress/blob/master/docker-compose.yml#L20-L21 ?